src/main/java/fi/aalto/cs/apluscourses/model/DummyExercise.java

```diff
@@ -0,0 +1,38 @@
+package fi.aalto.cs.apluscourses.model;
+
+import java.util.OptionalLong;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
+
+public class DummyExercise extends Exercise {
```

Comment:
Does this class actually differ from `Exercise`? It doesn't override anything. :thinking:

Reply:
i wanted to make a separate class to use instanceOf